keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -77,7 +82,14 @@ class StableDiffusion:
     - [Original implementation](https://github.com/CompVis/stable-diffusion)
     """
 
-    def __init__(self, img_height=512, img_width=512, jit_compile=False):
+    def __init__(
+        self,
+        img_height=512,
+        img_width=512,
+        input_img_height=512,
+        input_img_width=512,
```

Comment:
I don't love stacking these arguments here, but I also think asking for an image encoder in the constructor is a bit weird.

I suppose we could add a method `addImageEncoder` or `withImageEncoder`, but that feels a bit awkward also.

Reply:
How would a user initialize StableDiffusion with custom pre-trained weights for the VAE?

Reply:
@fchollet any thoughts on this?  I also dont love stacking the arguments like this.  Perhaps we just assume the input & output images are of static shape?

Regarding custom weights: you don't.  StableDiffusion is only mean't to replicate exact numerics of the checkpoint; nothing more, nothing less.

Reply:
Why not use the same size across the board?

Reply:
I guess theres no use case YET to use a dynamic input shape; but perhaps there may be one at some point?  

In any case, I have modified this.